app/views/employees/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<p style="color: green"><%= notice %></p>
```

Comment:
avoid to use inline styles as a good practice

Reply:
done